triton/ref/gpu.rst

```diff
@@ -2,19 +2,20 @@
-   NVIDIA V100          | ``gpu-v100-32g``                   | ``volta``                              | ``v100``                                | 40             | gpu[1-10]    | Volta          | 5120                      | 32GB              | 7.0
-   NVIDIA V100          | ``gpu-v100-32g``                   | ``volta``                              | ``v100``                                | 40             | gpu[28-37]   | Volta          | 5120                      | 32GB              | 7.0
-   NVIDIA V100          | ``gpu-v100-16g``                   | ``volta``                              | ``v100``                                | 176            | dgx[1-2,8-27]| Volta          | 5120                      | 16GB              | 7.0
-   NVIDIA V100          | ``gpu-v100-32g``                   | ``volta``                              | ``v100``                                | 32             | dgx[3,5-7]   | Volta          | 5120                      | 32GB              | 7.0
-   NVIDIA A100          | ``gpu-a100-80g``                   | ``ampere``                             | ``a100``                                | 56             | gpu[11-17,38-44] | Ampere     | 7936                      | 80GB              | 8.0
-   NVIDIA H100          | ``gpu-h100-80g``                   | ``hopper``                             | ``h100``                                | 16             | gpu[45-48]   | Hopper         | 16896                     | 80GB              | 9.0
-   NVIDIA H200          | ``gpu-h200-18g-ia``                | ``hopper``                             | ``h200-18g``                            | 56             | gpu[49]      | Hopper         |                           | 18GB              | 9.0
-   NVIDIA H200(*)       | ``gpu-h200-141g-ellis``, ``gpu-h200-141g-short``  | ``hopper``              | ``h200``                                | 112            | gpu[50-63]   | Hopper         |                           | 141GB             | 9.0
-   AMD MI100 (testing)  |   *Not yet installed*              | ``mi100``                              | Use ``-p gpu-amd`` only, no ``--gres``  |                | gpuamd[1]    |                |                           |                   |
+   GPU brand name       | GPU name in Slurm (``--gpus=NAME:n``) | Amount of VRAM                   | CUDA compute capability         | total amount   | nodes            | GPUs per node | Compute threads per GPU   | Slurm partition (``--partition=``)               |
+   NVIDIA H200(*)       | ``h200``                               | 141GB (``--gres=gpu-vram:141g``) | 9.0 (``--gres=min-cuda-cc=90``) | 112            | gpu[50-63]       | 8             | 16896                     | ``gpu-h200-141g-ellis``, ``gpu-h200-141g-short`` |
```

Comment:
I might leave the full command lines (``--gres=gpu-vram:141g``) out from this table and instead have another table below that goes into detail about the command line options.  We need special instructions that only the last one is used anyway.  The table can get quite long and right now the HTML doesn't scroll tables for some reason.